<launch>
    <node pkg="ros_pro2" type="controller_step3.py" name="controller" output="screen">
        <param name="linear_speed" value="0.2" />
        <param name="angular_speed" value="2"/>
        <param name="stop_distance" value="0.5"/>
        <param name="epsilon" value="0.01"/>
        
    </node>

    <node pkg="ros_pro2" type="monitor_step3.py" name="monitor"></node>
    <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_empty_world.launch">
        <arg name="x_pos" value="0.0"/>
        <arg name="y_pos" value="0.0"/>
        <arg name="z_pos" value="0.0"/>
    </include>
    
   <include file="$(find turtlebot_rviz_launchers)/launch/view_robot.launch"/>
</launch>
